packages/documentation/src/content/docs/integration/requirements/overview.mdx

```diff
@@ -64,6 +70,9 @@ You must integrate with an IdP if you plan to use the authorization server provi
 Ensure you've completed the following tasks before you deploy Rafiki to a production environment and join the Interledger network.
 
 - [ ] You are a licensed financial account servicing entity within the jurisdictions you operate in
+- [ ] You have generated a UUID v4 for `OPERATOR_TENANT_ID` and a strong, random `API_SECRET`
+- [ ] You have created the operator tenant record in the `backend` service's Postgres database with `OPERATOR_TENANT_ID` before first start
```

Comment:
The operator tenant record in the DB is seeded automatically, as long as `OPERATOR_TENANT_ID` and `API_SECRET` are defined